Form/View/ExportViewType.php

```diff
@@ -117,9 +138,32 @@ public function getBlockPrefix()
      */
     public function generateResponse(View $view, Request $request)
     {
-        $response = new Response();
         $parameters = $this->getParameters($view, $this->formFactory, $request);
-        
+
+        if (isset($view->getOptions()['export_type']) or $view->getOptions()['export_type'] === 'dompdf') {
+            // instantiate and use the dompdf class
+            $dompdf = new Dompdf();
+            $dompdf->loadHtml($parameters['render']);
+
+            // (Optional) Setup the paper size and orientation
+            $dompdf->setPaper(
+                (isset($view->getOptions()['pdf_size']) and $view->getOptions()['pdf_size']) ? $view->getOptions()['pdf_size'] : 'A4',
```

Comment:
preference for && instead of 'and' (convention?), as there is no precendence issue here